crates/bevy_ecs/src/query/filter.rs

```diff
@@ -219,6 +219,131 @@ unsafe impl<T: Component> WorldQuery for Without<T> {
 // SAFETY: no component access or archetype component access
 unsafe impl<T: Component> ReadOnlyWorldQuery for Without<T> {}
 
+/// A filter that tests if the given filter does not apply.
+/// This filter is intended to be used to determine whether a given component hasn't changed.
+///
+/// This filter does allow [`With`] or [`Without`] to be passed in. Instead, use their respective
```

Comment:
```suggestion
/// This filter does allow [`With`] or [`Without`] to be passed in, but may not work as expected. Instead, use their respective
```

Reply:
This was a typo of (does instead of does not). `Not<With<T>>` and `Not<Without<T>>` do not compile due to the associated type bound of `Item`.